scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+ id SERIAL PRIMARY KEY,
+ family_name TEXT,
+ given_name TEXT,
+ gender TEXT,
+ born_on DATE,
+ specialty TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
```

Comment:
This table is missing:
- `date` column for storing the appointment date.
- `time` column for storing the appointment time. 
- foreign key column referencing `doctors`.
- foreign key column referencing `patients`.

```
CREATE TABLE appointments(
  id SERIAL PRIMARY KEY,
  date DATE,
  time TIME,
  comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```